tests/cancun/eip5656_mcopy/test_mcopy_memory_expansion.py

```diff
@@ -40,14 +42,16 @@ def callee_bytecode(dest: int, src: int, length: int) -> Bytecode:
 
 
 @pytest.fixture
-def subcall_exact_cost(
+def call_exact_cost(
     initial_memory: bytes,
     dest: int,
     length: int,
 ) -> int:
     """
     Returns the exact cost of the subcall, based on the initial memory and the length of the copy.
     """
+    intrinsic_cost = 21000 + len(initial_memory) * 16
```

Comment:
```suggestion
    intrinsic_cost = 21000 + eip_2028_transaction_data_cost(initial_memory)
```
This will allow the data to be also zeros without issues (it can be imported from `ethereum_test_tools`).